src/pages/admin/AdminPage.scss

```diff
@@ -0,0 +1,176 @@
+@import '../../styles/_mixin';
```

Comment:
Did you use any mixin function in this css file, if not remove this import

Reply:
@papercoding Please review again ! 
<img width="1343" alt="Screen Shot 2020-09-27 at 19 50 30" src="https://user-images.githubusercontent.com/65656118/94365401-f4928880-00fa-11eb-983b-5667ca786fd7.png">